java/src/eda/sorting/Sorting.java

```diff
@@ -0,0 +1,10 @@
+package eda.sorting;
+
+/**
+ * Created by JoseManuel on 19/04/2016.
+ */
+public interface Sorting<T extends Comparable<T>> {
+    public static<T extends Comparable<T>> void sort(T[] arr){
```

Comment:
When you create a interface, the methods can't be statics. The interface methods can't be implemented.